src/module.c

```diff
@@ -861,6 +871,8 @@ RedisModuleCommand *moduleCreateCommandProxy(struct RedisModule *module, const c
  * * **"may-replicate"**: This command may generate replication traffic, even
  *                        though it's not a write command.
  * * **"no-mandatory-keys"**: All the keys this command may take are optional
+ * * **"allow-busy"**: Permit the command to run while another command is
```

Comment:
"another module command"

Reply:
technically, it'll also permit these module commands to run while there's a blocked script.
am i wrong?

Reply:
"another module command or a script"

Reply:
let's go with "Permit the command while the server is blocked either by a script or by a slow module command"
I feel it's important to document the scenarios in which this flags matters